src/skeleton.cpp

```diff
@@ -102,10 +102,11 @@ void init_skeleton(py::module & m) {
         })
 		.def("offset_polygons", [](const Skeleton_2_Ref &s, double offset) -> py::list {
 			py::list r;
-			auto result = CGAL::create_offset_polygons_2<Polygon_2>(offset, *s);
-			for (const auto &polygon : result) {
-				r.append(py::cast(new Polygon_2(*polygon), pybind11::return_value_policy::automatic));
-			}
+			// Commented, probably requires some form of conversion to work
```

Comment:
I guess if this cannot compile right now, it would be better to comment out the entire function?